File: app/src/main/java/com/iceteaviet/fastfoodfinder/data/remote/user/model/UserStoreEvent.java
package com.iceteaviet.fastfoodfinder.data.remote.user.model;

import com.iceteaviet.fastfoodfinder.data.remote.store.model.Store;

/**
 * Created by tom on 7/17/18.
 */
public class UserStoreEvent {
    public static final int ACTION_ADDED = 0;
    public static final int ACTION_CHANGED = 1;
    public static final int ACTION_REMOVED = 2;
    public static final int ACTION_MOVED = 3;

    private Store store;
    private int eventActionCode;

    public UserStoreEvent(Store store, int eventActionCode) {
        this.store = store;
        this.eventActionCode = eventActionCode;
    }

    public Store getStore() {
        return store;
    }

    public int getEventActionCode() {
        return eventActionCode;
    }
}

File: app/src/main/java/com/iceteaviet/fastfoodfinder/data/remote/user/model/UserStoreEvent.kt
package com.iceteaviet.fastfoodfinder.data.remote.user.model

import com.iceteaviet.fastfoodfinder.data.remote.store.model.Store

/**
 * Created by tom on 7/17/18.
 */
class UserStoreEvent(val store: Store, val eventActionCode: Int) {
    companion object {
        const val ACTION_ADDED = 0
        const val ACTION_CHANGED = 1
        const val ACTION_REMOVED = 2
        const val ACTION_MOVED = 3
    }
}
